src/components/forms/NetworkDevicesForm/InheritedNetworkRow.tsx

```diff
@@ -29,20 +30,38 @@ export const getInheritedNetworkRow = ({
   const isOverridden = overrideDevice !== undefined;
   const isDetached = overrideDevice && isNoneDevice(overrideDevice);
 
+  const deviceModified = overrideDevice && isDeviceModified(formik, device.key);
+  const initialOverrideDevice = formik.initialValues.devices.find(
+    (t) => t.name === device.key,
+  );
+  const overrideWasCleared = initialOverrideDevice && !overrideDevice;
+
+  const hasChanges = deviceModified || overrideWasCleared;
+
   const overrideNetwork = managedNetworks.find(
     (t) => t.name === (overrideDevice as LxdNicDevice)?.network,
   );
 
   return getConfigurationRowBase({
     configuration: (
-      <b
-        className={classnames({
-          "u-text--muted": isDetached,
-          "u-text--line-through": isDetached,
-        })}
-      >
-        {device.key}
-      </b>
+      <div className="network-device-name">
+        {hasChanges ? (
+          <span
+            className="network-device-change-indicator"
+            title="This inherited network has been modified"
+          ></span>
```

Comment:
```suggestion
          <span
            className="network-device-change-indicator"
            title="This inherited network has been modified"
          />
```
This can be self closing.